show-when-this.min.js

```diff
@@ -1,6 +1,6 @@
 /*!
-  show-when-this 0.1.2 - 19th Jan 2015
+  show-when-this 0.1.4 - 16th May 2015
   license: MIT
   @MikeRogers0
 */
-(function(e){"use strict";e.fn.showWhenThis=function(a){var r=e.extend({evaluator_callback:function(){return!0},change_selector:e("<div />"),_display_selector:e(this),_change_callback:function(){r.evaluator_callback()?r._display_selector.show():r._display_selector.hide(),r._display_selector.find("[data-required]:visible").prop("required",!0),r._display_selector.find("[data-required]:visible").attr("data-required",""),r._display_selector.find("[required]:hidden").attr("data-required","required"),r._display_selector.find("[required]:hidden").prop("required",!1)},_on_callback:"change"},a);r.change_selector.on(r._on_callback,r._change_callback),r.change_selector.trigger(r._on_callback)}})(window.jQuery||window.Zepto||window.$);
+!function(a){"use strict";a.fn.showWhenThis=function(b){var c=a.extend({evaluator_callback:function(){return!0},change_selector:a("<div />"),_display_selector:a(this),_change_callback:function(){c.evaluator_callback()?c._display_selector.show():c._display_selector.hide(),c._display_selector.find("[data-required]:visible").prop("required",!0),c._display_selector.find("[data-required]:visible").attr("data-required",""),c._display_selector.find("[required]:hidden").attr("data-required","required"),c._display_selector.find("[required]:hidden").prop("required",!1)},_on_callback:"change"},b);c.change_selector.on(c._on_callback,c._change_callback),c.change_selector.trigger(c._on_callback)}}(window.jQuery||window.Zepto||window.$);
```

Comment:
Line is too long.<br>Identifier 'evaluator_callback' is not in camel case.<br>Missing semicolon.<br>Identifier 'change_selector' is not in camel case.<br>Identifier '_display_selector' is not in camel case.<br>Identifier '_change_callback' is not in camel case.<br>Expected an assignment or function call and instead saw an expression.<br>Identifier '_on_callback' is not in camel case.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.